CLDConfig/Overlay/Overlay.py

```diff
@@ -39,7 +39,8 @@
         "HCalRingCollection", "380",
         "YokeBarrelCollection", "380",
         "YokeEndcapCollection", "380",
-        "LumiCalCollection", "380"
+        "LumiCalCollection", "380",
+        "ECalBarrelEta", "380",
```

Comment:
```suggestion
        "ECalBarrelEta", "380",  # for CLD_o4_v05 with LAr
```

Later to be separated in some way shape or form.